instances/widgets.treasury-factory.near/widget/pages/settings/systemupdates/UpdateRegistry.jsx

```diff
@@ -0,0 +1,11 @@
+return [
+  {
```

Comment:
ohh so going forward we will need to add all the updates in this widget, with required details, so users can update their widgets/contract?